### PR TITLE
LPS-177937 [EVP] Add comment to Approval or Rejection of organization's EVP Staff Approval on modal

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-177937](https://issues.liferay.com/browse/LPS-177937)

### CODE REVIEW

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:bundle

**Comment by juliocezar96** [comment]
ci:test:bundle

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js [hunk 1]
```diff
@@ -32,25 +33,49 @@ const updateStatus = async (key, name) => {
...
+	if (message === '') {
+		return attribute.removeAttribute('hidden');
+	}
+
+	return await updateStatus(key, value, message);
+};
+
 const openModal = () => {
+	let message = '';
+	let attributeHidden = '';
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js:46]:
```suggestion
	
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js [hunk 2]
```diff
@@ -32,25 +33,49 @@ const updateStatus = async (key, name) => {
...
 };
 
+const layerForDendingUpdateStatus = async (message, attribute, key, value) => {
+	if (message === '') {
+		return attribute.removeAttribute('hidden');
+	}
+
+	return await updateStatus(key, value, message);
+};
+
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js:43]:
```suggestion

const getMessage = () => document.querySelector('#messageDescribed').value;

const getAttributeHidden = () => document.querySelector('#messageDanger');

```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js [hunk 3]
```diff
@@ -32,25 +33,49 @@ const updateStatus = async (key, name) => {
...
 				},
 				type: 'submit',
 			},
 			{
 				displayType: 'success',
 				label: 'Approve',
 				async onClick() {
-					await updateStatus(
+					message = document.querySelector('#messageDescribed').value;
+					attributeHidden = document.querySelector('#messageDanger');
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js:75]:
```suggestion
				
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js [hunk 4]
```diff
@@ -32,25 +33,49 @@ const updateStatus = async (key, name) => {
...
 			{
 				displayType: 'danger',
 				label: 'Reject',
 				async onClick() {
-					await updateStatus('rejected', 'Rejected');
+					message = document.querySelector('#messageDescribed').value;
+					attributeHidden = document.querySelector('#messageDanger');
+					await layerForDendingUpdateStatus(
+						message,
+						attributeHidden,
```
Comment (eliasantosf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-organization-kebab-button/index.js:63]:
```suggestion
						getMessage(),
						getAttributeHidden(),
```

**Review by eliasantosf — CHANGES_REQUESTED** [review]

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @juliocezar96

**Comment by eliasantosf** [comment]
ci:forward

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:forward

**Pull request closed without merging**